pkg/crc/cluster/status.go

```diff
@@ -24,7 +24,13 @@ func WaitForClusterStable(ocConfig oc.Config, monitoringEnabled bool) error {
 			// update counter for consecutive matches
 			if status.IsReady() {
 				count++
+				if count == 1 {
+					logging.Info("All operators are available. Ensuring stability ...")
+				} else {
+					logging.Infof("Operators are stable (%d/%d) ...", count, numConsecutive)
+				}
 			} else {
+				logging.Info(status.String())
```

Comment:
I'd need to try it to see how this looks exactly, but this might be better as `Debug`?

Reply:
The goal is to show the user progress. This displays the lines:
```
INFO 3 operators are progressing: authentication, dns, image-registry 
INFO Operator image-registry is progressing       
INFO 2 operators are not available: authentication, openshift-apiserver 
```

Reply:
Clearliness (avoid confusion) is good. at this point, We can always decide to drop it if it becomes/feels like noise.